ansible/roles/airflow/tasks/main.yml

```diff
@@ -0,0 +1,155 @@
+---
+- set_fact:
+    svc_passwd:
+      - "{{ passwd.afredis }}"
+      - "{{ passwd.afpsql }}"
+      - "{{ passwd.afsched }}"
+      - "{{ passwd.afwebsrv }}"
+      - "{{ passwd.afworker }}"
+      - "{{ passwd.plpsql }}"
+      - "{{ passwd.plshovel}}"
+- group: name={{ item.login }} gid={{ item.id }} state=present
+  with_items: "{{ svc_passwd }}"
+- user:
+    name: "{{ item.login }}"
+    group: "{{ item.login }}"
+    uid: "{{ item.id }}"
+    comment: "{{ item.comment }}"
+    shell: /usr/sbin/nologin
+    createhome: no
+    home: /nonexistent
```

Comment:
Why? It's filled in passwd file anyway and existence of `/nonexistent` directory clearly shows that something is going wrong :)